golang1.15/lib/launcher.go

```diff
@@ -20,17 +20,26 @@ package main
 import (
```

Comment:
We don't need to backport to 1.15. We won't document it.

Reply:
Reverted 1.15 code back.